api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -55,6 +55,8 @@ const (
 	ClusterInstallationFailedMsg        string = "The installation has failed:"
 	ClusterInstallationNotStartedReason string = "InstallationNotStarted"
 	ClusterInstallationNotStartedMsg    string = "The installation has not yet started"
+	ClusterInstallationOnHoldReason     string = "InstallationOnHold"
+	ClusterInstallationOnHoldMsg        string = "The installation is on hold, to unhold set holdInstallation to false"
```

Comment:
For better grammar, I would make this two sentences. "The installation is on hold. To unhold set holdInstallation to false"
Write

Reply:
Sure, added documentation for the condition as well.